tests/e2e/login_test.go

```diff
@@ -41,18 +42,8 @@ var _ = Describe("odoLoginE2e", func() {
 			})
 		})
 
-		Context("Run login tests", func() {
+		Context("Run login tests with no active projects, having default is also considered as not having active project", func() {
 			AfterEach(func() {
-				projects := strings.Split(runCmd("oc projects -q"), "\n")
-				var waitOut bool
-				for _, p := range projects {
-					if len(p) > 0 {
-						waitOut = waitForCmdOut(fmt.Sprintf("%s %s", baseOdoProjectDelete, p), 10, func(out string) bool {
-							return strings.Contains(out, fmt.Sprintf("Deleted project : %s", p))
-						})
-						Expect(waitOut).To(BeTrue())
-					}
-				}
 				runCmd(backToCurrentUserCommand)
```

Comment:
What happens if no user was initially logged in? Wouldn't it be better to log the current user out? (Not sure, just asking)

Reply:
BeforeSuite actually ensures that.  It gets current project so a user has to be logged in, with an active project Infact

Reply:
Tests shouldn't really assume an initial state, otherwise they are not independent… but yes, it's probably better to leave the state how the tests found it so maybe, remember the previous user to be able revert to it but do a logout after that, so that the login tests can work without a user being logged in?

Reply:
Will do